### PR TITLE
6116 & 6111 - Various Holofuel Design

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: src/holofuel/components/SideMenu/SideMenu.js [hunk 1]
```diff
@@ -103,42 +76,24 @@ export function SideMenu ({
...
-    </div>
-    <div styleName='modal-buttons'>
-      <Button
-        onClick={handleClose}
-        styleName='modal-button-no'>
-        Ok
-      </Button>
-    </div>
-  </Modal>
-}
```
Comment (JettTech) [inline src/holofuel/components/SideMenu/SideMenu.js]: Was there a new decision to remove this modal?  The last time I was present to this conversation, we had agreed to leave the modal for the mobile interactions...

File: src/holofuel/components/SideMenu/SideMenu.js [hunk 2]
```diff
@@ -15,82 +13,57 @@ import {
...
-export function SideMenu ({
+export default function SideMenu ({
   isOpen,
   handleClose,
   avatarUrl = '',
   agent,
   agentLoading,
   inboxCount,
-  holofuelBalance,
-  isWide
```
Comment (JettTech) [inline src/holofuel/components/SideMenu/SideMenu.js:32]: Depending on the decision to render the Redeem HF modal, we might still need this 'isWide' prop.

File: src/holofuel/pages/Home/Home.module.css [hunk 3]
```diff
@@ -43,51 +45,82 @@
...
+.balance-row {
   display: flex;
   align-items: center;
 }
 
-.balance-label {
-  margin-left: auto;
-  font-size: 12px;
+.balance-amount {
+  text-align: center;
```
Comment (JettTech) [inline src/holofuel/pages/Home/Home.module.css:69]: The amount isn't actually centered.  Let's look into this...

File: src/holofuel/pages/Home/Home.js [hunk 4]
```diff
@@ -62,47 +64,50 @@ export default function Home () {
...
-    <div styleName='balance-and-transactions'>
-      <Link to={HISTORY_PATH} styleName='balance-link'>
-        <div styleName='balance'>
-          <div styleName='balance-header'>
-            <div styleName='balance-label'>
-              Available Balance
-            </div>
-            <div styleName='balance-arrow'>
-              >
+    <div styleName='container'>
```
Comment (JettTech) [inline src/holofuel/pages/Home/Home.js:72]: Let's add the `showAlphaFlag` prop here.

```
<PrimaryLayout headerProps={{ title: 'Home' }} showAlphaFlag={false}>
```

**Review by JettTech — CHANGES_REQUESTED** [review]
Great updates overall! :tada:  I have just a few comments/optimizations.

File: src/holofuel/components/SideMenu/SideMenu.js (hunk 1 above)
Comment (robbiecarlton) [reply src/holofuel/components/SideMenu/SideMenu.js]: I just slavishly followed the designs. There was no link to this in the designs.

File: src/holofuel/pages/Home/Home.module.css (hunk 3 above)
Comment (robbiecarlton) [reply src/holofuel/pages/Home/Home.module.css:69]: good catch, fixed.

File: src/holofuel/pages/Home/Home.js (hunk 4 above)
Comment (robbiecarlton) [reply src/holofuel/pages/Home/Home.js:72]: The designs have the Alpha Flag on the Home page.

https://www.figma.com/file/CCt10PCyyCVeo6G6T4j28O/HoloFuel?node-id=0%3A3681

File: src/holofuel/components/SideMenu/SideMenu.js (hunk 1 above)
Comment (robbiecarlton) [reply src/holofuel/components/SideMenu/SideMenu.js]: I've asked C and Z to see what they think.

**Pull request merged**